packages/lesmis-server/src/routes/ReportRoute.ts

```diff
@@ -26,20 +27,23 @@ export class ReportRoute extends Route {
     switch (type) {
       case 'dashboard': {
         if (legacy === 'true') {
-          const legacyReport = await this.webConfigConnection.fetchDashboardReport({
+          const legacyReport = await this.webConfigConnection.fetchDashboardReport(reportCode, {
```

Comment:
A bit awkward here because web-config-server report endpoint will also check for `legacy`. But I'm not too fussed about it 😬